llarp/router/router.cpp

```diff
@@ -925,15 +913,17 @@ namespace llarp
       // the whitelist enabled and we got the whitelist
       // check against the whitelist and remove if it's not
       // in the whitelist OR if there is no whitelist don't remove
-      if (has_whitelist and not _rc_lookup_handler.is_session_allowed(rc.router_id()))
+      if (not node_db()->is_connection_allowed(rc.router_id()))
```

Comment:
Bringing the `rc_lookup_handler` functions into `NodeDB` is pretty clean

Reply:
They had to live somewhere, and rc_lookup_handler is thankfully obsolete.  It was spaghetti from the moment I wrote it. :'(